GoogleForCreators/web-stories-wp
# Code Quality: Remove out of date dashboard preview experiment

## Context

Once upon a time last summer, we had a dashboard experiment to preview a story/template in the dashboard (via a dialog) instead of opening a new tab. We've since decided against this because of the clutter it adds to the UI and the ambiguous way we'd have to load the content - it just doesn't add enough value to keep it.

## Summary

Guts the story preview code in the dashboard as well as deletes the 2 experiments related to previews (`enableStoryPreviews` and `enableTemplatePreviews`).

## Relevant Technical Choices

Just deleting code.

## To-do



## User-facing changes

If you had all experiments in the dashboard turned off then you'll see no changes. 

For our internal team's staging environment which has all the experiments on, you'll notice that `preview` buttons on stories are gone in the dashboard and `preview` is no longer an option in the context menus for stories in the grid and the list view. 

## Testing Instructions

Using staging as a sanity check for the experiment states is a good idea for this. 
The middle button that is visible on hover of a story should now be gone. (the button should remain for templates since that takes you to the detail view). 

Now, if you enable ` Enable in-progress story actions` experiment and open a context menu for a story, you won't see the 'preview' option anymore. You can still open the story in a new tab, just not in a dialog. 

### QA

Regression testing around the dashboard to make sure nothing was broken residually. 




This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #6962
